error-hierachy.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
What if we don't create a new file, but share a single file across all languages? 
I'd prefer to make this particular definition transpileable, because we already can transpile nested dicts of dicts easily. We'd want to avoid disk io at all costs.

Reply:
https://github.com/ccxt/ccxt/blob/master/js/base/errors.js